src/sql/parser.ts

```diff
@@ -6,6 +6,16 @@ import type { SqlParseError, SqlParseResult, SqlParser } from "./types.js";
 
 interface ParserOption extends Option {
   database: SupportedDialects;
+  /**
+   * If true, the parser will quote brackets in the SQL query which will satisfy the parser.
+   * This is useful if you want to interpolate variables in f-strings.
+   *
+   * @example
+   * ```sql
+   * SELECT {id} -> SELECT '{id}'
```

Comment:
this _might_ change the col-number of errors that appear elsewhere. happy to do that in followup if you'd prefer since might be a bit tricky to account for